### PR TITLE
Add the emptystate layout to weblinks

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue # .

### Summary of Changes
If no weblink exists, the empty stat screen appeasr as in other components

Need help for a good and correct text on the empty stat screen. COM_WEBLINKS_EMPTYSTATE_CONTENT
This is why I make it as a draft. 

### Testing Instructions
Make a fresh installation of weblinks or delete all weblinks. 

grafik

### Expected result
The empty state screen appears. 

### Actual result
The overvies appears with a information.

### Documentation Changes Required
yes

### CODE REVIEW

File: src/language/en-GB/com_weblinks.ini [hunk 1]
```diff
@@ -11,6 +11,9 @@ COM_WEBLINKS_DEFAULT_PAGE_TITLE="Web Links"
 COM_WEBLINKS_EDIT="Edit Web link"
 COM_WEBLINKS_EDIT_PUBLISHED_WEBLINK="Published Weblink"
 COM_WEBLINKS_EDIT_UNPUBLISHED_WEBLINK="Unpublished Weblink"
+COM_WEBLINKS_EMPTYSTATE_BUTTON_ADD="Add your first Web Link"
+COM_WEBLINKS_EMPTYSTATE_CONTENT="A Web Link .. please help ...."
```
Comment (joomdonation) [inline src/language/en-GB/com_weblinks.ini:15]: @brianteeman When you have a moment, could you please help with this empty state text ?

Comment (brianteeman) [reply src/language/en-GB/com_weblinks.ini:15]: add, edit and remove links to other web sites on your Joomla! web site, and organize them into categories. You can then display these links on your site, and optionally let visitors add new links.

Comment (joomdonation) [reply src/language/en-GB/com_weblinks.ini:15]: Many thanks for your help.

Comment (chmst) [reply src/language/en-GB/com_weblinks.ini]: Thanks, @brianteeman

**Pull request closed without merging**